### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
## In this PR:
- I created a new branch  named `forms`
- I added create post and create comment form as required
- I ran linter validation check
- I created a pull request

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,14 +1,20 @@
-<header>
 <%= stylesheet_link_tag 'user', media: 'all', 'data-turbolinks-track': 'reload' %>
-</header>
+<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
 <body>
+
+<nav class="nav">
+    <%= link_to "Add New Post", new_user_post_path(@user.id) %>
+    <%= link_to "Home", root_path %>
```
Comment (Davidosky007) [inline app/views/users/show.html.erb:7]: * Kindly fix this error i get when i try to test your forms below 👇 is a recording of the error 

https://github.com/Peter-Dumbari/Blog-Application/assets/52497246/eb525b03-6f02-4c80-bd67-361c6d4a6691

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Peter-Dumbari 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* No linter errors 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
-</div>
\ No newline at end of file
+</div>
+
+<div class="like-comment"><%= render "comments/create" %> 
+<%if Like.where(author_id: User.first.id, post_id: @post.id).exists?%>
+ <p id= "liked">Liked<p>
+<%else%>
+<%= render "likes/create" %></div>
+<%end%>
```
Comment (codepantha) [inline app/views/posts/show.html.erb:28]: - [ ] An error occurs when attempting to create a new post and it has to do with a missing partial `likes/_create`as shown below 👇. As a result of this bug, I am unable to test the comment and like functionality. Please try to fix this error 🐛 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @Peter-Dumbari 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Highlights 🎯 
✔️ Descriptive PR
✔️ Added forms to create new posts and comments 📌 
✔️ No linter errors 🟢 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 2 above)
Comment (Peter-Dumbari) [reply app/views/posts/show.html.erb:28]: @codepantha, sorry how are you starting the app?

**Review by franclobo — APPROVED** [review]
Hi @Peter-Dumbari, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Good UI ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Peter-Dumbari** [comment]
@franclobo, Thanks I appreciate 🙇‍♂️

**Pull request merged**